packages/sdk38/src/cosmosclient.searchtx.spec.ts

```diff
@@ -104,20 +105,22 @@ describe("CosmosClient.searchTx", () => {
           },
         };
         const transactionId = await client.getIdentifier(tx);
-        const heightBeforeThis = await client.getHeight();
         try {
           await client.postTx(tx.value);
         } catch (error) {
           // postTx() throws on execution failures, which is a questionable design. Ignore for now.
```

Comment:
What alternative would you suggest?

I think it is good to ensure people don't blindly ignore failures (if they don't read the exection result), but it should be easy to detect ("tx ran but failed in checktx/delivertx" along with the log info from the chain)

Reply:
Do you want to create an issue for this approach and we can discuss this?

I would not change it here, but for sdk39 I think we will change lots of interfaces, so good to check all issues with the current design so we can improve on them,

Reply:
Okay, I put it here: https://github.com/CosmWasm/cosmwasm-js/issues/199
There was https://github.com/CosmWasm/cosmwasm-js/issues/194 already which shows the current solution is unsufficient.